site/content/docs/main/file-system-backup.md

```diff
@@ -64,6 +64,11 @@ for namespace2 would be `https://s3-us-west-2.amazonaws.com/bucket/kopia/ns2`.
 There may be additional installation steps depending on the cloud provider plugin you are using. You should refer to the 
 [plugin specific documentation](supported-providers.md) for the must up to date information.  
 
+**Note:** Currently, Velero creates a secret named `velero-repo-credentials` in the velero install namespace, containing a default backup repository password.
+You can update the secret with your own password encoded as base64 prior to the first backup (i.e., FS Backup, data mover) targeting to the backup repository. The value of the key to update is `Data[repository-password]`.
```

Comment:
This is cherry-pick but I think long term (another PR?) we would want this to be a yaml example rather than Golang map.
````suggestion
You can update the secret with your own password encoded as base64 prior to the first backup (i.e., FS Backup, data mover) targeting to the backup repository. The value of the key to update is
```
data:
  repository-password: <custom-password>
```
````

Reply:
cc: @shubham-pampattiwar

Reply:
@kaovilai If we do want that, it should go into main

Reply:
This is just the release-branch PR.

Reply:
https://github.com/vmware-tanzu/velero/pull/6783